SimCalorimetry/EcalSimProducers/plugins/PreMixingEcalWorker.cc

```diff
@@ -70,8 +71,9 @@ PreMixingEcalWorker::PreMixingEcalWorker(const edm::ParameterSet &ps,
       m_EEs25notCont(ps.getParameter<double>("EEs25notContainment")),
       m_peToABarrel(ps.getParameter<double>("photoelectronsToAnalogBarrel")),
       m_peToAEndcap(ps.getParameter<double>("photoelectronsToAnalogEndcap")),
-      theEBSignalGenerator(EBPileInputTag_, tok_eb_, m_EBs25notCont, m_EEs25notCont, m_peToABarrel, m_peToAEndcap),
-      theEESignalGenerator(EEPileInputTag_, tok_ee_, m_EBs25notCont, m_EEs25notCont, m_peToABarrel, m_peToAEndcap),
+      m_timeDependent(ps.getUntrackedParameter<bool>("timeDependent", false)),
```

Comment:
This parameter should be tracked (affects physics). Since the parameter is premixing-specific, it should be sufficient to add it to this PSet
https://github.com/cms-sw/cmssw/blob/b93fe86e012255468ada74ae052bc4da24204323/SimGeneral/PreMixingModule/python/mixOne_premix_on_sim_cfi.py#L75-L90